tsup.config.ts

```diff
@@ -0,0 +1,59 @@
+import { defineConfig } from 'tsup'
+
+export default defineConfig({
+  entry: [
+    "./src/index.ts",
+    "./src/interceptors/ClientRequest/index.ts",
+    "./src/interceptors/fetch/index.ts",
+    "./src/interceptors/XMLHttpRequest/index.ts"
+  ],
+  outDir: "./lib"
```

Comment:
yep I think so

Reply:
What format do you want this outputted in exactly?

For example, it can look like this:
![Screenshot 2022-12-02 at 11 24 37](https://user-images.githubusercontent.com/20640274/205282260-866c1893-59e3-406d-89e4-cd87ba89f288.png)

Or like this (tsup's "legacy output"):
![Screenshot 2022-12-02 at 11 25 43](https://user-images.githubusercontent.com/20640274/205282465-29ca4c06-e9b4-4014-949b-0b3e9d9a9270.png)

I really don't know enough about ESM to judge what's best here